cmd/erasure-healing.go

```diff
@@ -1019,6 +1032,22 @@ func isAllNotFound(errs []error) bool {
 	return len(errs) > 0
 }
 
+// isAllBucketsNotFound will return true if all the errors are either errFileNotFound
+// or errFileCorrupt
+// A 0 length slice will always return false.
+func isAllBucketsNotFound(errs []error) bool {
+	if len(errs) == 0 {
+		return false
+	}
+	notFoundCount := 0
+	for _, err := range errs {
+		if err != nil && (err.Error() == errFileNotFound.Error() || err.Error() == errFileCorrupt.Error()) {
```

Comment:
So isAllBucketsNotFound() takes errs from readAllBucketInfo();

readAllBucketInfo() runs xl-storage.StatVol(), which returns errVolumeNotFound, not errFileNotFound; 

errFileCorrupt is not returned as well, so no need to check for it

Reply:
ah yes, that's a mistake. Will change to use `errVolumeNotFound` and remove `errFileCorrupt`